paparazzi-gradle-plugin/src/test/projects/common.gradle

```diff
@@ -0,0 +1,20 @@
+// TODO:
+//   Remove when https://github.com/google/guava/issues/6567 is fixed.
+//   See also: https://github.com/google/guava/issues/6801.
+plugins.withId("app.cash.paparazzi") {
+  // Defer until afterEvaluate so that testImplementation is created by Android plugin.
+  afterEvaluate {
+    dependencies.constraints {
+      add("testImplementation", "com.google.guava:guava") {
+        attributes {
+          attribute(
+            TargetJvmEnvironment.TARGET_JVM_ENVIRONMENT_ATTRIBUTE,
+            objects.named(TargetJvmEnvironment, TargetJvmEnvironment.STANDARD_JVM)
+          )
+        }
+        because("LayoutLib and sdk-common depend on Guava's -jre published variant." +
+          "See https://github.com/cashapp/paparazzi/issues/906.")
+      }
```

Comment:
Could/should this be somehow done in a way that it affects all of Paparazzi's users? If someone uses Paparazzi on `testImplementation`, they'll likely run into issues (we have so many reported in this repo).

Reply:
I can't think of a way, without force bundling Guava/Truth, which I'm reluctant to do.  @JakeWharton, any thoughts?